share/ci/compiler_hipcc.yml

```diff
@@ -6,8 +6,7 @@
   image: registry.hzdr.de/crp/alpaka-group-container/alpaka-ci-${CI_CONTAINER_NAME}-rocm${HIP_CONTAINER_VERSION}-pic:${CONTAINER_TAG}
   variables:
     GIT_SUBMODULE_STRATEGY: normal
-    # ISAAC is not working with HIP
-    DISABLE_ISAAC: "yes"
+    DISABLE_ISAAC: "no"
```

Comment:
Could you please remove this line fully? We have places in the scripts where we check only if the variable is set. IN this case we do not use ISAAC.
IMO ISAAC is still disabled and therefore all passed the CI.

Reply:
@psychocoderHPC done